karma.config.js

```diff
@@ -1,20 +1,18 @@
-//? our webpack config imports as a function, and running it returns the required object to be used below
-const webpackConfig = require('./webpack.config.js')(null, { mode: 'test' });
-// clear entry file config (if exists)
-webpackConfig.entry = () => ({});
+const webpackConfig = require('./config/webpack.test.js')();
 
 module.exports = config => {
   config.set({
     basePath: '',
 
     // frameworks: https://npmjs.org/browse/keyword/karma-adapter
-    frameworks: ['jasmine'],
+    frameworks: ['jasmine', 'webpack'],
 
     // This will be the new entry to webpack so it should just be a single file
-    files: ['src/index.tests.js'],
+    files: ['src/main.js', 'src/index.tests.js'],
 
     // Preprocess test index and test files using webpack (will run babel)
     preprocessors: {
+      'src/main.js': ['webpack'],
       'src/index.tests.js': ['webpack'],
       'src/**/*.test.js': ['webpack'],
```

Comment:
This line shouldn't be needed. The `*.test.js` files are processed as part of `index.tests.js` afaict.

Reply:
To clarify, I am commenting on line 17 (`src/**/*.test.js`), not line 15 (`src/main.js`) :grin: